po/es.po

```diff
@@ -5550,6 +5742,9 @@ msgid ""
 "we wish to concisely express more complex ideas than patterns alone would "
 "allow."
 msgstr ""
+"Los guardias de coincidencia, como característica sintáctica independiente, "
```

Comment:
"Las guardas de coincidencia...."

https://dle.rae.es/guarda

In Spanish "guarda" can be used to refer as a protection to something, for example when placing white sheets of paper at the top and bottom of a printer article to protect it; it is also used for the sword guard.